templates/common/.devcontainer/devcontainer.json/csharp/devcontainer.json

```diff
@@ -7,11 +7,19 @@
         }
     },
     "features": {
-        "github-cli": "2",
-        "azure-cli": "2.38",
-        "dotnet": "6.0",
-        "docker-from-docker": "20.10",
-        "node": {
+        "ghcr.io/devcontainers/features/github-cli:1": {
+            "version": "2"
+        },
+        "ghcr.io/devcontainers/features/azure-cli:1": {
+            "version": "2.38"
+        },
+        "ghcr.io/devcontainers/features/docker-from-docker:1": {
+            "version": "20.10"
+        },
+        "ghcr.io/devcontainers/features/dotnet:1": {
+            "version": "6.0"
+        },
+        "ghcr.io/devcontainers/features/node:1": {
```

Comment:
would it make sense to sort these entries alphabetically?  (just for readability/discoverability? )

Reply:
Sure! At first I thought there was some sort of specificity ordering (i.e. target language last) but it wasn't consistent. Alphabetical is the easiest to be consistent about. I can change that shortly.